docs/pages/server-access/guides/ssh.mdx

```diff
@@ -0,0 +1,215 @@
+
+This Guide describes common scenarios one is likely to encounter when using Teleport in such a way.
+
+<Admonition type="warning" title="Warning">
+  The examples below may include the use of the `sudo` keyword to make following each step easier when creating resources from scratch. However, we generally discourage the use of `sudo` in real-world production environments per the *Principle of Least Privilege* (POLP). 
+</Admonition>
+
+## SSH mode and adding an SSH Node to your cluster
+
+Teleport [Nodes](../../architecture/nodes.mdx) have three modes: *Auth*, *Proxy*, and *Node* (SSH) mode. A single Node can run in up to all three modes. Optionally, you can configure a Node to act as the dedicated *Auth* or SSH Node for a cluster.
+
+<Figure
+  align="center"
+  bordered
+  caption="Teleport SSH agent forwarding."
+>
+  ![Example cluster arrangement](../../../img/server-access/teleport-ssh.svg)
+</Figure>
+
+### Configuration
+
+The two primary ways a Node can be configured into SSH (or any) mode is through a `teleport.yaml` configuration file or via the `sudo tctl tokens add` command which accepts the `--type` parameter to specify which of `auth`, `proxy`, or `node` (default) the Node will be automatically configured as.
+
+In both cases, you'll likely use a *static* (fixed) or *dynamic* (valid for a limited duration of time) [join token](../../admin-guide.mdx#adding-nodes-to-the-cluster) to link your Nodes to existing clusters.
+
+To link an SSH Node with an existing `auth.example.com` cluster:
+
+1. Execute `sudo tctl tokens add --type=node` on the `auth` cluster machine.
+
+   - Save the resultant `token` value to a token-specific directory of your choice. For example: `/tmp/token`. 
+   - This helps to minimize the direct sharing of tokens even when they are *dynamically* generated.
+
+2. Your console will display the following message format: 
+
+   ```bash
+   # Join cluster
+   sudo teleport start \
+     --roles=node \
+     --token=./token.file \
+     --auth-server=tele.example.com:443
+   ``` 
+
+3. Run the command above with the correct `auth.example.com` public proxy address on the machine you wish to connect.
+
+   - This second machine (and its Node) need not publically exposed (per the best-practice *Bastion pattern*).
+
+4. You should now be able to view both Nodes in the Teleport Web Interface after logging in with your user.
+
+   <Figure
+     align="center"
+     bordered
+     caption="Both Nodes in the Web UI"
+   >
+     ![Both Nodes in the Web UI](../../../img/server-access/teleport_ui.png)
+   </Figure>
+
+## Connect to a cluster using tsh ssh
+
+Teleport SSH is robust enough that you can do all the things you'd do with OpenSSH, without it.
+
+To connect to a cluster (`auth.example.com`) with public proxy `auth.example.com:443` as user `auth-admin` using `tsh ssh`:
```

Comment:
When do we create `auth-admin`?  and is `access` and `editor` enough permissions for that user?

Reply:
Thanks that's a great point - I've rewritten that part to include more recent items from the Getting Started document. This is mostly temporary staging for moving a lot of content around. Links break, images are shifting, etc.